scan/lib/scan/detect_values.rb

```diff
@@ -175,10 +175,26 @@ def self.detect_destination
 
       # building up the destination now
       if Scan.devices && Scan.devices.count > 0
```

Comment:
Seems right on the surface level, but unfortunately I have [further concerns](https://github.com/fastlane/fastlane/pull/5159#issuecomment-271700480).